timesketch/ui/explore/filter.directive.ts

```diff
@@ -68,6 +68,7 @@ export const tsFilter = function () {
                 scope.meta.es_total_count = 0
                 scope.meta.es_time = 0
                 scope.meta.noisy = false
+                ctrl.search(scope.query, scope.filter, scope.queryDsl)
```

Comment:
I'll have to ask you more about this later.  I don't really know what this does.

Reply:
This is how AngularJS 1.x works. Happy to explain more later.